iterative/utils/analytics.go

```diff
@@ -350,10 +345,13 @@ func JitsuEventPayload(action string, e error, extra map[string]interface{}) (ma
 		"os_name":      systemInfo["os_name"],
 		"os_version":   systemInfo["platform_version"],
 		"backend":      extra["cloud"],
-		"error":        err,
 		"extra":        extra,
 	}
 
+	if e != nil { 
+		payload["error"] = reflect.TypeOf(e).String()
```

Comment:
Yes, the goal is providing some information about the error type but not exfiltrating sensitive information

Reply:
ah, that would be an absolutely brilliant comment to add at line 352 :)

Reply:
It looks like we have some brilliant review comments at https://github.com/iterative/terraform-provider-iterative/pull/493#discussion_r858523852 😄

Reply:
```suggestion
		// Include the type of the error as a string, but not the error message; it can be sensitive
		payload["error"] = reflect.TypeOf(e).String()
```

Reply:
couldn't resist

```suggestion
		// only use error type (not message) to avoid potentially sensitive info
		payload["error"] = reflect.TypeOf(e).String()
```